resources/views/admin/addCategory.blade.php

```diff
@@ -12,10 +12,9 @@
             <h3 class="mb-3">Add category</h3>
             <h5>Title</h5>
             <input id="title" type="text" class="form-control{{ $errors->has('title') ? ' is-invalid' : '' }} mb-4 col-12" name="title" value="{{ old('title') }}" required>
```

Comment:
In the database, the maximum number of characters you can input for the title is only 191. You should add a "maxlength" attribute on your input element to restrict the number of characters entered. Also do this for "edit category". Any input exceeding 191 characters for title would result to an error in you current implementation.